Description: Don't try running binaries for objcopy tests
Author: Stephen Kitt <skitt@debian.org>

When cross-compiling in an environment that can run target binaries, a
number of the objcopy tests fail. This patch forces the target
detection to skip running target binaries.

--- a/upstream/binutils/testsuite/binutils-all/objcopy.exp
+++ b/upstream/binutils/testsuite/binutils-all/objcopy.exp
@@ -702,6 +702,8 @@
 	return 2
     }
 
+    return 3
+
     set result [remote_load target tmpdir/$test_prog]
     set status [lindex $result 0]
 
